service_thing_aggregate/src/main/scala/controllers/ObjectSearchController.scala

```diff
@@ -0,0 +1,49 @@
+package controllers
+
+import com.google.inject.Inject
+import models.MusitThing
+import no.uio.musit.service.MusitResults.{MusitDbError, MusitError, MusitResult, MusitSuccess}
+import play.api.Logger
+import play.api.libs.concurrent.Execution.Implicits.defaultContext
+import play.api.libs.json.{JsValue, Json}
+import play.api.mvc.{Action, Controller, Result, Results}
+import services.ObjectSearchService
+
+/**
+ * Created by jarle on 05.10.16.
+ */
+
+object Utils {
```

Comment:
Well, the thing is that we cannot say that there will be a standard way of doing this. Because each controller method may decide for itself what to do in case of success or error. Assuming `Results.Ok`, or `Results.InternalServerError` will only cover a subset of use-cases. We have tests to ensure that the results are returned as expected.